dependency_graph/README.md

```diff
@@ -0,0 +1,10 @@
+
+To generate the `dependancy_graph.svg` the executables `tred` and `dot` are required from the
```

Comment:
I am talking about `dependancy_graph.svg`, I think it should be `dependency_graph.svg`.